drainer/server.go

```diff
@@ -86,9 +86,15 @@ func init() {
 
 // NewServer return a instance of binlog-server
 func NewServer(cfg *Config) (*Server, error) {
-	ID, err := genDrainerID(cfg.ListenAddr)
-	if err != nil {
-		return nil, errors.Trace(err)
+	var ID string
```

Comment:
how about remove the `ID`, and use cfg.NodeID directly

Reply:
Revised.